lib/iris/tests/test_cell.py

```diff
@@ -130,15 +130,15 @@ class Terry:
         self.assertEqual(self.d.__ne__(Terry()), NotImplemented)
 
     def test_numpy_int_equality(self):
-        dtypes = (np.int, np.int16, np.int32, np.int64)
+        dtypes = (int, np.int16, np.int32, np.int64)
         for dtype in dtypes:
             val = dtype(3)
             cell = iris.coords.Cell(val, None)
             self.assertEqual(cell, val)
 
     def test_numpy_float_equality(self):
         dtypes = (
-            np.float,
+            float,
```

Comment:
Given that this method name contains 'numpy' I'm not sure if a bare `float` should remain here now.

Reply:
```suggestion
            np.float_,
```

Reply:
@akuhnregnier Using `np.float_` might be a reasonable substitute here :+1: